src/components/projectSelect/index.tsx

```diff
@@ -129,13 +128,8 @@ export const ProjectSelect: React.VFC<ProjectSelectProps> = ({
   );
 };
 
-const { reportError } = errorReporting;
-
 export const getRepoId = (projects: Project[]) => {
   if (!projects || projects.length === 0) {
-    reportError(
-      new Error("No projects provided when fetching repo id")
-    ).warning();
```

Comment:
This was throwing warnings on every render, but it's not actually an error if a repo exists with no projects attached to it.